# FUYU Gantry Slider Control 

int32 pos_x1
int32 pos_x2
int32 pos_y

int32[] err_code
